protos/route.proto

```diff
@@ -88,6 +88,7 @@ message Script {
   string source = 4;
 }
 message GetScriptsRequest {
+  int32 id = 1;
```

Comment:
would probably be cleaner to create a separate endpoint, but doesn't really matter